src/Hosting/WindowsServices/src/WindowsServiceLifetimeHostBuilderExtensions.cs

```diff
@@ -26,6 +26,22 @@ public static class WindowsServiceLifetimeHostBuilderExtensions
         /// <param name="hostBuilder">The <see cref="IHostBuilder"/> to operate on.</param>
         /// <returns>The same instance of the <see cref="IHostBuilder"/> for chaining.</returns>
         public static IHostBuilder UseWindowsService(this IHostBuilder hostBuilder)
+        {
+            return UseWindowsService(hostBuilder, _ => { });
```

Comment:
Maybe just pass `null` as action down, then check in the other overload for `!= null`?

Reply:
We do this pattern a lot, so I'd suggest whatever we do in other places like this :). I *think* we do it this way (passing in an empty delegate), but I don't have a concrete example handy.

Reply:
In a quick search I didn't find an example either. I prefer consistency too.

Reply:
I think I'd keep it as an empty delegate. It seems to be the thing we generally do in these situations. There may be a super-small perf impact (calling an empty delegate vs a null check) but this is during start-up so I'm not super worried.

I'd rather just keep `null`s out of our code as much as possible, so this code as-is seems good to me.